website/static/js/issue.js

```
@@ -168,3 +168,515 @@ $(function () {
+            
+            // Get issues for the current page
+            const url = `https://api.github.com/repos/${repo}/issues?state=all&per_page=${perPage}&page=${page}`;
+            const response = await fetch(url);
+            
+            if (!response.ok) throw new Error('GitHub API request failed');
+            
+            const issues = await response.json();
+            
+            // Check if there are more pages
+            const linkHeader = response.headers.get('Link');
+            hasMorePages = linkHeader && linkHeader.includes('rel="next"');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)